cmd/exchange_wrapper_standards/exchange_wrapper_standards_test.go

```diff
@@ -382,6 +376,7 @@ func generateMethodArg(ctx context.Context, t *testing.T, argGenerator *MethodAr
 			Description:   "1337",
```

Comment:
On line 601 there is a section on unsupported exchanges, please remove coinbasepro from there now that you've updated it all. Plus it passes once removed 🎉